### ISSUE TITLE
New RTINGS FR target

### ISSUE THREAD

**Issue by Kevorex** [jaakkopasanen/AutoEq#182]
Could anyone explain why we need a different target as opposed the one the RTINGS.com used to review their headphones? I was merely a user who grabs the convolver IRs without thanking Mr. Pasanen, but now I'm here to say a big thank you<3 for creating all these hard work.

**Comment by NekoAlosama** [comment]
From the README.md:
`All of the results use frequency response targets that were specifically developed for this project except oratory1990 and Crinacle's IEM measurements which use standard Harman targets. The target curves were developed by calibrating measurements against reference measurements by oratory1990 and Crinacle (IEMs) and modifying the Harman 2018 over-ear and 2019 in-ear targets with the calibration data.`

In other words, this means that the [target curve](https://github.com/jaakkopasanen/AutoEq/blob/master/measurements/rtings/resources/rtings_harman_over-ear_2018_wo_bass.png) used on Rtings measured headphones is about equal to the Harman curves, which is preferred over [Rtings' target curve.](https://github.com/jaakkopasanen/AutoEq/blob/master/measurements/rtings/resources/rtings_compensation_avg.png)

**Comment by Kevorex** [comment]
Oh I see, there was a lot of changes here. I was used to the old InnerFidelity to Serious on SBAF target to calibrate my Philips X2HR (well, the HR is just a rebrand with that Japanese Hi-Res certification thingy), and I always thought X2HR got a big treble spike at 9k that's hissing so much. Now that X2HR has been reviewed by RTINGS, and when I look at the three Oratory1990, RTINGS, and InnerFidelity to Harman target results for the X2HR, they all showed that the 9k spike is actually caused by InnerFidelity and RTINGS's measurement setups rather than X2HR is actually too loud at 9k. This is totally mind-blowing, but now I find calibrations on this project a bit too harsh on 9k lol.
